lib/wibox/widget/imagebox.lua

```diff
@@ -182,6 +182,9 @@ function imagebox:draw(ctx, cr, width, height)
 
     local w, h = self._private.default.width, self._private.default.height
 
+    local RsvgRect = Rsvg.Rectangle {x = 0, y = 0, width = w, height = h}
```

Comment:
what if image is not an svg though?

Reply:
Right, I could simply move it to where I apply it and just `or` it. I'll do it some time this week together with part 2 for the recoloring container

Reply:
my point was that it seems that one of unit tests related to images is failing when invoked with non-svg image:

https://github.com/awesomeWM/awesome/actions/runs/4834063772/jobs/8614833487?pr=3802